tmeister/static/src/release-notes/release-notes-table/release-note-inline-preview.component.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { md } from '../release-notes.helper.js'
+import { IconButton, Icon } from '@material-ui/core'
+
+const useStyles = makeStyles(theme => ({
+  flex: {
+    display: 'flex'
+  }
+}))
+
+export default function ReleaseNoteInlinePreview ({markdown = ''}) {
+  const [inlinePreview, setInlinePreview] = useState(true)
+  const classes = useStyles()
+  let preview = markdown
+  let showArrow = false
+  if (markdown && markdown.length > 5) {
+    const split = markdown.split('\n')
+    showArrow = split.length > 5
+    if (inlinePreview && showArrow) {
+      const slice = split.slice(0, 5)
+      slice.push('...')
+      preview = slice.join('\n')
+    }
+  }
+  const html = md.render(preview)
```

Comment:
I can eliminate the need for a custom hook here because I know we need to re-do all of this whenever the markdown prop changes.